compose/cli/command.py

```diff
@@ -25,6 +25,7 @@
 def project_from_options(project_dir, options):
     override_dir = options.get('--project-directory')
     environment = Environment.from_env_file(override_dir or project_dir)
+    environment.silent = options['COMMAND'] == 'exec'
```

Comment:
Perhaps, instead of `environment.silent = options['COMMAND'] == 'exec'`
to avoid `KeyErrors` consider:
 `environment.silent = options.get('COMMAND', 'exec') == 'exec'`

Reply:
Hmmm... I agree about avoiding `KeyErrors`, but I would use:
`environment.silent = options.get('COMMAND', None) == 'exec'`

Since we would like to silence it if and only if it's an `exec` command.
WDYT @mmphego ?

Reply:
Fully agree.